index.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Dunk card display</title>
+		<link rel="stylesheet" type="text/css" href="quize.css">
+	</head>
+	<body>
+
+		<article1 class="box">
```

Comment:
So requirement 1 specifies that each card should be an `<article>`, you are creating a custom tag called article1, article2, article3, and article4. This works (as in doesn't break), but we want each card to be the same default article.